### ISSUE TITLE
Set "Include in VR Library" for VR games automatically

### ISSUE THREAD

**Issue by Bluscream** [bburky/playnite-non-steam-shortcuts#8]
<details>
<summary>Screenshot(s)</summary>

![](https://cdn.discordapp.com/attachments/699577276571975760/710953951326896168/image.png)</details>

The formular i would suggest using:

```py
(game.name.lower().contains(" VR ")) ||
(game.name.lower().endswith("VR")) ||
(game.name.lower().startswith("VR")) ||
(game.source == "Oculus") ||
(game.source == "Viveport") ||
(str(game.tags).lower().contains("VR"))
```

**Comment by bburky** [comment]
A similar and simpler option is to require all games to be placed in a category called "VR". If a game is in the VR category it would be marked as openvr in the Steam shortcut.

Do you still want `allowoverlay` enabled on VR games? Not sure what that actually does for VR games. Are these non-SteamVR games you're launching from Steam shortcuts, or SteamVR games you're launching? I wonder if that matters for the overlay?

**Comment by Bluscream** [comment]
> A similar and simpler option is to require all games to be placed in a category called "VR". If a game is in the VR category it would be marked as openvr in the Steam shortcut.
As much as i would love this approach, it would not work with games from oculus for example. All their games are VR games so they don't bother with returning "VR" as a tag or feature

> Do you still want allowoverlay enabled on VR games? Not sure what that actually does for VR games. Are these non-SteamVR games you're launching from Steam shortcuts, or SteamVR games you're launching? I wonder if that matters for the overlay?

I think it doesn't matter. Overlay is handled by the vrdashboard while in VR which is active anyway while steam is running

**Comment by bburky** [comment]
Yeah, I think I set the category manually. I like auto setting based on the source. But parsing the name of the game seems a bit questionable.

**Comment by Bluscream** [comment]
Sure, i can always set that later in the source myself!
